src/index.js

```diff
@@ -208,6 +208,8 @@ class Firestore {
     });
 
     this._validator = new Validator({
+      ArrayTransform: (name, value) =>
```

Comment:
This results in an `isArrayTransform()` function that actually validates that a value can be used _in_ an array transform, not that it _is_ an array transform.

Should this be `ArrayTransformElement`?  Or is this generic enough (since you can pass options to allow/disallow deletes, etc.) that it should just be `FieldValue` ?